MvvmCross/Core/Core/Platform/LogProviders/ConsoleLogProvider.cs

```diff
@@ -179,6 +179,9 @@ private static class ConsoleColorValues
             static ConsoleColorValues()
             {
                 Type = Type.GetType("System.ConsoleColor");
+                if (Type == null)
```

Comment:
Please revert this.

Reply:
On my System everytime I used Type = Type.GetType("System.ConsoleColor") the result was null. It was working with Type.GetType("System.ConsoleColor, System.Console"); Still want me to revert?